src/encoding.js

```diff
@@ -317,7 +317,7 @@ function addGlyphNamesAll(font) {
         const c = charCodes[i];
         const glyphIndex = glyphIndexMap[c];
         glyph = font.glyphs.get(glyphIndex);
-        glyph.addUnicode(parseInt(c));
+        glyph && glyph.addUnicode(parseInt(c));
```

Comment:
you already used the optional chaining `.?` syntax early on,
so keep it unified (your choice :) )

Reply:
Oh, did I use optional chaining? I thought that would throw an error with the current reify setup... I'll take a look at it!

Reply:
Yup here:

https://github.com/opentypejs/opentype.js/pull/704/files#diff-cd810153d07820a50b3950a985c42117e1dd7ea75207520e60b7306ce17b8901R396